Sources/Html/View.swift

```diff
@@ -27,3 +27,13 @@ extension View where M == [Node] {
     self.call = const([node])
   }
 }
+
+extension View where A == (), M == [Node] {
```

Comment:
nice nice, but I believvveeeee I ran into problems with these overloads :( we'll soon see!

maybe write some tests to cover some common use cases too...